opal/mca/common/ofi/common_ofi.c

```diff
@@ -340,10 +340,8 @@ static uint32_t get_package_rank(opal_process_info_t *process_info)
                                        &pname, &locality_string, PMIX_STRING);
         if (PMIX_SUCCESS != rc || NULL == locality_string) {
             // If we don't have information about locality, fall back to procid
-            opal_output_verbose(1, opal_common_ofi.output,
-                                "%s:%d:Unable to get locality string from local peers.\n"
-                                "This may negatively impact performance.\n",
-                                __FILE__, __LINE__);
+            if (9 < opal_output_get_verbosity(opal_common_ofi.output))
+		opal_show_help("help-common-ofi.txt", "package_rank failed", true);
```

Comment:
Can you put `{}` around all blocks, even 1-line blocks?

Also, you might want to do something like this to help prevent the text of the message from getting stale:

```c
int level = 10
if (opal_output_get_verbosity(opal_common_ofi.output) >= level) {
    opal_show_help("help-common-ofi.txt", "package_rank failed", true, level);
}
```

And then change your addition to the help message to be:

```
Note: This message is displayed only when the OFI component's verbosity level is 
%d or higher.
```